README.md

```diff
@@ -341,9 +341,15 @@ This error occurs in environments where the standard
 [`crypto.getRandomValues()`](https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues)
 API is not supported. This issue can be resolved by adding an appropriate polyfill:
 
-| Environment  | Suggested Polyfill                                                                                  |
-| ------------ | --------------------------------------------------------------------------------------------------- |
-| React Native | [`react-native-get-random-values`](https://github.com/LinusU/react-native-get-random-values#readme) |
+### React Native
+
+1. Install [`react-native-get-random-values`](https://github.com/LinusU/react-native-get-random-values#readme)
+1. Import it before `uuid` ([example](./examples/reactnative/)):
+
+```javascript
+import 'react-native-get-random-values';
+import { v4 as uuidv4 } from 'uuid';
+```
```

Comment:
I liked the way nanoid documented this: https://github.com/ai/nanoid/blob/master/README.md#react-native

I'm hoping that giving a code example inline in this readme will save us some useless issues where people mess up import order just because the don't check out the example in detail.